include/retdec/fileformat/file_format/elf/elf_format.h

```diff
@@ -23,6 +24,11 @@ namespace fileformat {
 class ElfFormat : public FileFormat
 {
 	private:
+		std::vector<std::string> telfhashSymbols;
+		/// flag if we already loaded symbols from SHT_DYNSYM
+		bool telfhashDynsym = false;
```

Comment:
Again, could we maybe get rid of this somehow? If not then ok, but these flags are often hard to maintain, it would be better to determine this somehow from the result data themselves, or something else.

Reply:
I didn't find a nicer way to keep the state, because of the way the symbol loading works, I must hold it through multiple functions calls of `loadSymbols()`. I guess static variable in the function works too, but I am not sure if that is a better solution tho.